templates/mempoolTx.html

```diff
@@ -0,0 +1,122 @@
+                        <i class="fa fa-copy"></i>
+                      </button>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class="row border-bottom p-3 mx-0">
+                <div class="col-md-3">Status:</div>
+                <div class="col-md-9">
+                  <div class="d-flex flex-wrap">
+                    <div class="mr-2 flex-shrink-1">
+                      <h5 class="m-0">
+                        <span class="badge badge-info badge-pill align-middle text-white"><i class="fas fa-info-circle pr-1"></i> In Mempool</span>
+                      </h5>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class="row border-bottom p-3 mx-0">
+                <div class="col-md-3">From:</div>
+                <div class="col-md-9">{{ formatEth1AddressFull .From }}</div>
+              </div>
+              <div class="row border-bottom p-3 mx-0" style="border-width:4px !important;">
+                <div class="col-md-3">{{ if .IsContractCreation }}Creates Contract{{ else if .TargetIsContract }}Interacts With{{ else }}To{{ end }}:</div>
+                {{ if .To }}
+                  <div class="col-md-9">
+                    <div class="d-flex">
+                      <div class="row no-gutters w-100">
+                        <div class="col-md-auto" style="min-width: 0;">
+                          <span class="align-middle mr-2">{{ formatEth1AddressFull .To }}</span>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                {{ end }}
+              </div>
+              <div class="row border-bottom p-3 mx-0">
+                <div class="col-md-3">Value:</div>
+                <div class="col-md-9">{{ formatBigAmount .Value "ETH" 8 }}</div>
+              </div>
+              <div class="row border-bottom p-3 mx-0">
+                <div class="col-md-3">Gas Limit:</div>
+                <div class="col-md-9">{{ formatBigNumberAddCommasFormated .Gas 0 }}</div>
+              </div>
+              <div class="row border-bottom p-3 mx-0" style="border-width:4px !important;">
+                <div class="col-md-3">Gas Fee Cap:</div>
+                <div class="col-md-9">{{ formatBigAmount .GasFeeCap "GWei" 5 }}</div>
+              </div>
+              <div class="row border-bottom p-3 mx-0">
+                <div class="col-md-3">Attributes:</div>
+                <div class="col-md-9">
+                  <div class="d-flex flex-wrap">
+                    <div class="mr-2 flex-shrink-1">
+                      <span class="badge badge-dark align-middle text-light p-1 px-2">Nonce: <span class="text-white">{{ bigToInt .Nonce }}</span></span>
```

Comment:
When you view a tx that has no `.TransactionIndex` and no `.Input`, the last field shown is `Attributes`. It looks like this:
![image](https://user-images.githubusercontent.com/114066135/222081426-f5c4b957-e54c-4823-ab0f-83ef7e64765b.png)

The thin line between `Attributes` and the table's border should be removed imho.